index.js

```diff
@@ -1,6 +1,7 @@
 var domToReact = require('./lib/dom-to-react');
 var attributesToProps = require('./lib/attributes-to-props');
 var htmlToDOM = require('html-dom-parser');
+var { Element } = require('domhandler/lib/node');
```

Comment:
Since this file isn't compiled, can you update so it doesn't destructure?

Reply:
Sure, thanks for the feedback. I can do that for sure.

Reply:
See https://github.com/remarkablemark/html-react-parser/pull/296/commits/f57d7da7e15e0e71ed2bf69a4a0219d8c5b5adfc, please.